aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -53,7 +53,7 @@ Tensor& _clamp__cpu(Tensor& self, optional<Scalar> min, optional<Scalar> max) {
   } else if (min) {
     return _th_clamp_min_out(self, self, *min);
   } else {
-    return self;
```

Comment:
can `_clamp__cpu` be written as:
```
return _clamp_out_cpu(self, self, *min, *max);
```

Reply:
Good point

Reply:
I changed it so that _clamp__device usees clamp_out_device